### DISCUSSION TITLE
[Question]: Deletion of chats by user

### DISCUSSION THREAD

**Discussion by k-toki** [danny-avila/LibreChat#1381] [Q&A]
### What is your question?

Is it possible to prohibit users from deleting chats or disable the delete button?

### More Details

None

### What is the main subject of your question?

Documentation

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
No, that's a bad design pattern against GDPR so I won't be working on this. You're free to fork and edit the code as you like, though
